exec/java-exec/src/test/java/org/apache/drill/test/BaseTestQuery.java

```diff
@@ -119,6 +125,15 @@ public static void setupDefaultTestCluster() throws Exception {
     // turns on the verbose errors in tests
     // sever side stacktraces are added to the message before sending back to the client
     test("ALTER SESSION SET `exec.errors.verbose` = true");
+    emptyDirCreating();
```

Comment:
Why do we create empty directory for all tests that extend this class? I guess need to create it only for those tests that need it.